REVIEWING.md

```diff
@@ -0,0 +1,119 @@
+Disagreements between maintainers can generally be worked out during PR review through comments.
+If this is not possible, other maintainers can be called on to give their opinions and determine a way forward.
+We encourage such disagreements and discussions, so long as they remain respectful and are done with the goal of resolving the dispute.
+
+If a decision cannot be reached, the issue may be put to a vote, in which all Maintainers of the repository in question and all Core Maintainers can vote.
+
+Being respectful includes respect for the contributor's time.
+If there is a disagreement, please do not make the contributor make repeated changes until an agreement on how to proceed is reached.
+Also, please attempt to reach an agreement quickly, so the PR can be merged in a timely fashion.
+
+### Language Version Updates
+
+Most repositories in the Podman project are written in Go, and as such target a specific version of Go in their `go.mod`.
+For example:
+```
+$ cat go.mod | grep 'go 1.'
+go 1.22.8
+```
+
+For all branches except the main branch, this should always remain static.
+PRs into a non-main branch which change the supported Go version should be modified to not require such a change, or rejected if that is not possible.
+Changing supported Go version in the main branch is allowed, but not encouraged.
+Doing so will require significant discussion among maintainers, as it can affect what distributions the main branch can be built.
+
+### Tests
+
+Please check tests added by the PR.
+If a PR has no new tests, determine if this is actually appropriate - has new functionality been added which should have been tested?
+If the PR does have tests, check to see if they are reasonably comprehensive.
+The Podman Project does not have code coverage standards at present, but we aim to ensure that all new functionality is tested.
+Tests for bugs and new functionality should, generally speaking, fail when run against Podman without the patch applied.
+Reviewers are encouraged to check this when reviewing a pull request.
+
+### Documentation
+
+All changes to public-facing APIs (e.g. the Podman REST API) and CLI should be appropriately documented.
+API changes require Swagger documentation.
+CLI changes should be documented in the Manpages.
+
+Please validate that PRs making such changes include appropriate documentation.
+Many repositories in the Podman project will enforce this via CI check, but you should still review the contents of the documentation to make sure they are appropriate and complete.
+
+## Things to Avoid
+
+### Bikeshedding and excessively critical reviews
+
+Please avoid bikeshedding during reviews.
+
+Trivial changes that do not affect the ultimate functionality of the PR - for example, small grammar or phrasing changes in documentation, unnecessary renaming of variables - should not block merge of a PR.
```

Comment:
I agree with non-blocking PRs for internal code comments, but I disagree with anything that is customer-facing, whether it's output to be read by a customer or, more importantly, the man pages. Those errors in documentation should be blockable, IMO.

Reply:
Would be good to make spellcheck or other tools a part of CI (I think we already do it in some projects). While they need not be PR blockers, I'd say it reflects poorly if we have simple typos in documentation. While we're at it, maybe we should also stick with a variant of English unless chosen already. For example, Fedora [enforces American English](https://docs.fedoraproject.org/en-US/packaging-guidelines/ReviewGuidelines/#_things_to_check_on_review): 
```
MUST: The spec file must be written in American English. See Packaging Guidelines: Summary and description
```